conf/layer.conf

```diff
@@ -11,6 +11,9 @@ BBFILE_PRIORITY_raspberrypi = "9"
 
 LAYERSERIES_COMPAT_raspberrypi = "nanbield scarthgap"
 LAYERDEPENDS_raspberrypi = "core"
+# Recommended for u-boot support for raspberrypi5
+# https://git.yoctoproject.org/meta-lts-mixins 'scarthgap/u-boot' branch
+LAYERRECOMMENDS_raspberrypi = "lts-u-boot-mixin"
```

Comment:
Why isn't this updated in oe-core?

Reply:
You cannot do a major bump to oe-core for the scarthgap release. It was during the feature freeze. So oe-core will forever be at v2024.01, which does not have raspberrypi5 support.

Reply:
I understand that for scarthgap. But this is targeting the master branch. I would expect this layer recommends on a scarthgap branch backport. Am I missing something obvious?

Reply:
> I understand that for scarthgap. But this is targeting the master branch. I would expect this layer recommends on a scarthgap branch backport. Am I missing something obvious?

There is no scarthgap branch in meta-raspberrypi, so that cannot be targeted for a pull request.

A backport to oe-core scarthgap stable branch will never be accepted, it is a major u-boot upgrade. It breaks stable release policy.

Reply:
Thanks, @moto-timo. Sparkling clear to me. Looks good!